pkg/zhonghong_protocol/zhonghongapi.go

```diff
@@ -0,0 +1,7 @@
+package zhonghongprotocol
+
+type Client interface {
+	// Bit access
+
+	ReadGateway() (results []byte, err error)
```

Comment:
please add a todo here. Long term wise the results should be a struct instead of []byte